packages/client/test/get.spec.js

```diff
@@ -49,6 +49,15 @@ describe('get', () => {
     assert.ok(res.ok)
     const files = await res.files()
     assert.is(files.length, 3, 'should contain 3 files')
+    assert.is(files[0].name, 'dr-is-tired.jpg')
```

Comment:
we should not depend on network requests in CI, but let's deal with this later

Reply:
is a local mock 🎉

Reply:
https://github.com/web3-storage/web3.storage/blob/a95bcb2f7d8a2164cc6ef7e90b7d72bb1b0f096a/packages/client/test/mocks/api/get_car%23%40cid.js#L5-L10